src/20240201_AaveV3BNB_OnboardFdUSDToAaveV3OnBSC/OnboardFdUSDToAaveV3OnBSC.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Onboard fdUSD to Aave v3 on BSC"
```

Comment:
```suggestion
title: "Onboard fdUSD to Aave v3 on BNB"
```
it's super confusing because they rebranded and it's still called bscscan, but in fact correct name is BNB